mysql/tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}-{5.7,8.0,maria10.2,maria10.5}
+    py{27,38}-{5.7,8.0,8.0-group,maria10.2,maria10.5}
```

Comment:
It's supported, but since we already have 10 different mysql environment I prefer testing one group-replication only so the tests aren't too long. The additional value of testing both `5.7-group` and `8.0-group` is pretty low since we already test classic `5.7` and `8.0`